crates/vm/levm/src/vm.rs

```diff
@@ -67,6 +67,95 @@ impl StateBackup {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+/// This structs holds special configuration variables specific to the
+/// EVM. In most cases, at least at the time of writing (February
+/// 2025), you want to use the default blob_schedule values for the
+/// specified Fork. The "intended" way to do this is by using the `EVMConfig::canonical_values(fork: Fork)` function.
+///
+/// However, that function should NOT be used IF you want to use a
+/// custom ForkBlobSchedule, like it's described in
+/// EIP-7840(https://eips.ethereum.org/EIPS/eip-7840). For more
```

Comment:
You probably missed it, but adding this will add the link to "EIP-7840" when the docstring is parsed.
```suggestion
/// [EIP-7840](https://eips.ethereum.org/EIPS/eip-7840). For more
```

Reply:
Oh! My bad. I did miss it. Thanks!